Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,19 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+import CardBanner from "./CardBanner";
+import CardContent from "./CardContent";
+
+const CardContainer = () => {
+  return (
+    <a class="link" href="https://www.reactjs.org">
+      <div className="card-container-full">
+        <div className="card-container">
+          <CardBanner />
+          <CardContent />
+        </div>
+      </div>
```

Comment:
this looks like another case of an unnecessary div since the only child of div.card-container-full is another div